ftplugin/elixir.vim

```diff
@@ -51,3 +51,19 @@ silent! setlocal formatoptions-=t formatoptions+=croqlj
 
 let b:undo_ftplugin = 'setlocal sw< sts< et< isk< com< cms< path< inex< sua< def<'.
       \ '| unlet! b:match_ignorecase b:match_words b:block_begin b:block_end'
+
+function s:projectionist_detect_elixir()
+  if filereadable('mix.exs')
+    let root = simplify(fnamemodify('.', ':p:s?[\/]$??'))
+
+    let projections = {}
+    let projections['lib/*.ex'] = { 'type': 'main', 'alternate': ['test/{}_test.exs'] }
+    let projections['lib/**/*.ex'] = { 'type': 'main', 'alternate': ['test/{}_test.exs'] }
```

Comment:
Are these additional `lib/**/` and `test/**/` entries necessary? The `lib/*.ex` / `test/*.ex` entries work well for me, even with file hierarchies.

Reply:
No, these aren't needed, quoting docs:

> From a globbing perspective, `"*"` is actually a stand in for `"**/*"`.  For advanced cases, you can include both globs explicitly: `"test/**/test_*.rb"`. When expanding with `{}`, the `**` and `*` portions are joined with a slash.  If necessary, the `dirname` and `basename` expansions can be used to split the value back apart.